Formula/swift-format.rb

```diff
@@ -26,14 +25,14 @@ class SwiftFormat < Formula
   # out of the box on Xcode-only systems due to an incorrect sysroot.
   pour_bottle? only_if: :clt_installed
 
-  depends_on xcode: ["14.0", :build]
+  depends_on xcode: ["14.2", :build]
 
   uses_from_macos "swift"
 
   def install
     # Support Swift 5.7.
     # Remove when minimum supported Swift >= 5.7.1.
-    inreplace "Package.swift", '.upToNextMinor(from: "0.50700.0")', '.exact("0.50700.0")' if OS.mac? && build.stable?
+    inreplace "Package.swift", '.upToNextMinor(from: "0.50700.1")', '.exact("0.50700.1")' if OS.mac? && build.stable?
```

Comment:
```suggestion
```

Reply:
I'm neither swift package manager or home-brew expert. Could you elaborate a little on why this is helpful?
Per the comment above the line, this should be removed when swift is updated, but I didn't explicitly update swift. Maybe it was implicitly updated with the Xcode toolchain update?

Reply:
No, it says it can be removed when Swift 5.7 is supported on swift-format.